Bug: WASM build fails after f54be9a with unresolved reinhardt::commands and tokio imports

## Description

After updating reinhardt-web to commit `f54be9a5`, WASM target compilation (`wasm32-unknown-unknown`) fails with multiple errors:

1. `reinhardt::commands` - gated behind `not(target_arch = "wasm32")` but still imported
2. `reinhardt::reinhardt_auth` - gated behind `not(target_arch = "wasm32")` but still referenced 
3. `auth::models` - gated behind `#[cfg(native)]` but still referenced
4. `tokio` - unresolved in wasm32 target

These errors appear when `runserver --with-pages` triggers the WASM build for the Pages SPA.

## Steps to Reproduce

1. Create a reinhardt-web project with `pages` feature enabled
2. Run `cargo run --bin manage runserver 0.0.0.0:8000 --with-pages`
3. Observe WASM compilation errors

## Expected Behavior

WASM target compilation should succeed as it did with previous commits (before f54be9a5).

## Actual Behavior

```
error[E0432]: unresolved import `reinhardt::commands`
error[E0433]: failed to resolve: could not find `models` in `auth`
error[E0432]: unresolved import `reinhardt::reinhardt_auth`
error[E0433]: failed to resolve: use of unresolved module or unlinked crate `tokio`
```

## Environment

- reinhardt-web: f54be9a5
- Rust: stable
- Target: wasm32-unknown-unknown
- OS: macOS

🤖 Generated with [Claude Code](https://claude.com/claude-code)

--- a/examples/examples-tutorial-basis/Cargo.toml
+++ b/examples/examples-tutorial-basis/Cargo.toml
@@ -20,7 +20,7 @@ serde_json = "1.0.145"
 
 # WASM-specific dependencies
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-reinhardt = { workspace = true, features = ["pages"] }
+reinhardt = { workspace = true, features = ["pages", "client-router"] }
 wasm-bindgen = "0.2.106"
 wasm-bindgen-futures = "0.4.56"
 web-sys = { version = "0.3.83", features = [

--- a/examples/examples-tutorial-basis/src/config/urls.rs
+++ b/examples/examples-tutorial-basis/src/config/urls.rs
@@ -4,9 +4,11 @@
 
 #[cfg(server)]
 use reinhardt::pages::server_fn::ServerFnRouterExt;
+#[cfg(server)]
 use reinhardt::prelude::*;
 #[cfg(server)]
 use reinhardt::routes;
+use reinhardt::UnifiedRouter;
 
 // Import server_fn marker modules (snake_case + ::marker)
 #[cfg(server)]

--- a/examples/examples-twitter/src/apps/auth/shared/types.rs
+++ b/examples/examples-twitter/src/apps/auth/shared/types.rs
@@ -4,6 +4,7 @@
 //! These types are serializable and can be sent between the WASM client
 //! and the Rust server via server functions.
 
+#[cfg(server)]
 use reinhardt::Validate;
 use serde::{Deserialize, Serialize};
 use uuid::Uuid;
@@ -37,36 +38,47 @@ impl From<crate::apps::auth::models::User> for UserInfo {
 
 /// Login request
 #[cfg_attr(server, derive(Schema))]
-#[derive(Debug, Clone, Serialize, Deserialize, Validate)]
+#[cfg_attr(server, derive(Validate))]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct LoginRequest {
-	#[validate(email(message = "Invalid email address"))]
+	#[cfg_attr(server, validate(email(message = "Invalid email address")))]
 	pub email: String,
 
-	#[validate(length(min = 1, message = "Password is required"))]
+	#[cfg_attr(server, validate(length(min = 1, message = "Password is required")))]
 	pub password: String,
 }
 
 /// Register request
 #[cfg_attr(server, derive(Schema))]
-#[derive(Debug, Clone, Serialize, Deserialize, Validate)]
+#[cfg_attr(server, derive(Validate))]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct RegisterRequest {
-	#[validate(length(
-		min = 3,
-		max = 150,
-		message = "Username must be between 3 and 150 characters"
-	))]
+	#[cfg_attr(
+		server,
+		validate(length(
+			min = 3,
+			max = 150,
+			message = "Username must be between 3 and 150 characters"
+		))
+	)]
 	pub username: String,
 
-	#[validate(email(message = "Invalid email address"))]
+	#[cfg_attr(server, validate(email(message = "Invalid email address")))]
 	pub email: String,
 
-	#[validate(length(min = 8, message = "Password must be at least 8 characters"))]
+	#[cfg_attr(
+		server,
+		validate(length(min = 8, message = "Password must be at least 8 characters"))
+	)]
 	pub password: String,
 
-	#[validate(length(
-		min = 8,
-		message = "Password confirmation must be at least 8 characters"
-	))]
+	#[cfg_attr(
+		server,
+		validate(length(
+			min = 8,
+			message = "Password confirmation must be at least 8 characters"
+		))
+	)]
 	pub password_confirmation: String,
 }
 

--- a/examples/examples-twitter/src/apps/profile/shared/types.rs
+++ b/examples/examples-twitter/src/apps/profile/shared/types.rs
@@ -3,6 +3,7 @@
 //! These types are serializable and can be sent between the WASM client
 //! and the Rust server via server functions.
 
+#[cfg(server)]
 use reinhardt::Validate;
 use serde::{Deserialize, Serialize};
 use uuid::Uuid;
@@ -40,17 +41,24 @@ impl From<crate::apps::profile::models::Profile> for ProfileResponse {
 
 /// Update profile request
 #[cfg_attr(server, derive(Schema))]
-#[derive(Debug, Clone, Serialize, Deserialize, Validate, Default)]
+#[cfg_attr(server, derive(Validate))]
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct UpdateProfileRequest {
-	#[validate(length(max = 500, message = "Bio must be less than 500 characters"))]
+	#[cfg_attr(
+		server,
+		validate(length(max = 500, message = "Bio must be less than 500 characters"))
+	)]
 	pub bio: Option<String>,
 
-	#[validate(url(message = "Invalid avatar URL"))]
+	#[cfg_attr(server, validate(url(message = "Invalid avatar URL")))]
 	pub avatar_url: Option<String>,
 
-	#[validate(length(max = 100, message = "Location must be less than 100 characters"))]
+	#[cfg_attr(
+		server,
+		validate(length(max = 100, message = "Location must be less than 100 characters"))
+	)]
 	pub location: Option<String>,
 
-	#[validate(url(message = "Invalid website URL"))]
+	#[cfg_attr(server, validate(url(message = "Invalid website URL")))]
 	pub website: Option<String>,
 }

--- a/examples/examples-twitter/src/apps/tweet/shared/types.rs
+++ b/examples/examples-twitter/src/apps/tweet/shared/types.rs
@@ -3,6 +3,7 @@
 //! These types are serializable and can be sent between the WASM client
 //! and the Rust server via server functions.
 
+#[cfg(server)]
 use reinhardt::Validate;
 use serde::{Deserialize, Serialize};
 use uuid::Uuid;
@@ -67,12 +68,16 @@ impl From<crate::apps::tweet::models::Tweet> for TweetInfo {
 
 /// Create tweet request
 #[cfg_attr(server, derive(Schema))]
-#[derive(Debug, Clone, Serialize, Deserialize, Validate)]
+#[cfg_attr(server, derive(Validate))]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct CreateTweetRequest {
-	#[validate(length(
-		min = 1,
-		max = 280,
-		message = "Tweet must be between 1 and 280 characters"
-	))]
+	#[cfg_attr(
+		server,
+		validate(length(
+			min = 1,
+			max = 280,
+			message = "Tweet must be between 1 and 280 characters"
+		))
+	)]
 	pub content: String,
 }

